### PR TITLE
adding troubleshooting section to readme

### PR DESCRIPTION
**Pull request by karlw00t**
This might not be the correct way to do this, but I wanted to throw an idea out there.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -53,6 +53,16 @@ Installation
 
        $ pip install aws-encryption-sdk
 
+Troubleshooting
+===============
+
+Ensure you have the most recent version of pip on your system:
```
Comment (mattsb42-aws) [inline README.rst]: It's not directly about having the most recent version of pip but about having the most recent version of `pyca/cryptography` and `cffi`. Getting past a certain version of each of those requires a minimum version of pip.

Comment (juneb) [reply README.rst]: This info should be in the README. Tell people when they can prevent a problem. Matt, if you can write a bullet point or two, I'll make it into words for the readme.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 *********
-Changelog
+Changelo
 *********
 
-1.3.7 -- 2018-09-20
+1.3.7 -- 2018-09-2001
```
Comment (mattsb42-aws) [inline CHANGELOG.rst]: ? That is not a valid date.

**Comment by acioc** [comment]
Closing stale PR, please reopen as needed

**Pull request closed without merging**